www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -195,7 +204,8 @@ function (declare, Deferred, ObjectStore, Store, QueryResults, topic,
         { name : 'Message', field : 'checkerMsg', width : '100%' },
         { name : 'Checker name', field : 'checkerId', cellClasses : 'link', width : '50%' },
         { name : 'Severity', field : 'severity' },
-        { name : 'Suppress', field : 'suppressComment' }
+        { name : 'Review status', field : 'reviewStatusHtml', cellClasses : 'review-status', width : '25%' },
+        { name : 'Review comment', field : 'reviewComment' }
```

Comment:
Do we really wish to show the full comment on the bug list? What if the comment gets too long?

Perhaps we should instead only show a "message" icon or something clickable that presents the comment as a pop-up?
Or implement a way to "crop" the too long text at least, because this happens: 
![image](https://user-images.githubusercontent.com/1969470/29026479-3bdcceb0-7b7c-11e7-8eec-8084ac8fed57.png)